.rubocop.yml

```diff
@@ -12,7 +12,7 @@ AllCops:
 ClassLength:
   Max: 150
 Layout/LineLength:
-  Max: 120
+  Max: 160
```

Comment:
Is it be possible to wrap any long lines instead of bumping the rubocop line length here? 160 sounds a little unreadable locally.

Reply:
I tried at the time but it didn't work without it. However it would make sense that there is, so I will look at it again.

Reply:
From what I'm seeing, I can either have lines too long, or methods too long. I think the lesser of two evils would be to have longer lines for better readability.

Reply:
There we go, fixed with most recent commit.